plugins/Satellites/src/Satellites.hpp

```diff
@@ -74,7 +74,7 @@ By convention, group names are in lowercase. The GUI translates some of the grou
 
 <b>Satellite Catalog</b>
 
-The satellite catalog is stored on disk in [JSON](http://www.json.org/) format, in a file named "satellites.json". A default copy is embedded in the plug-in at compile time. A working copy is kept in the user data directory.
+The satellite catalog is stored on disk in [JSON](http://www.json.org/) format, in a file named "satellites.json". A default copy is embedded in the plug-in at compile time. A working copy is kept in the user data directory. Test for GitHub diff, are lines now soft wrapped?
 
```

Comment:
@gzotti here the soft wrap does work fines in GitHub diff (split mode, but apparently not in unified mode). nevertheless IMHO we should avoid hard newlines, as most editors offer soft wrap anyway.

In fact, hard newlines are a thing of the past in the era of 4k screens and intelligent editors... GitHub should not lead to this kind of limitations.

And otherwise at some point someone will have to edit away hard line breaks because lines become too fragmented.

Reply:
I see the line upto "... in a file nam" in the box above. It is cumbersome enough to move the scroller (which is available in the box above), but most online diff views don't provide a scroll bar, making it extra nasty to move around. Yes, I have a wide screen, but GH's style only uses 1/3 of it. And others don't have wide screens. 

Line length is something that may be regarded as part of the project code style, similar to e.g. indentation or bracketing rules. It is not only for Github. Our style guide. Defined by the maintainers and forced upon the heads and fingers of all contributors before a maintainer will press "Merge". Please adhere to the code style when not instructed otherwise. Resistance is futile.

Reply:
ok!

Reply:
by the way, try shift+scroll while in the diff pane.

(and remember: life = permanent evolution, permanent re-questioning of existing values, permanent refactoring. be agile.)